packages/babel-traverse/src/scope/index.js

```diff
@@ -830,7 +832,7 @@ export default class Scope {
         return scope;
       }
     } while ((scope = scope.parent));
-    throw new Error("We couldn't find a Function or Program...");
+    return null;
```

Comment:
to keep both consistent, what do you think of not throwing in both cases and returning `null`? 
(although i'm letting scope's `getProgramParent()` throw since that would mean something is weird?)